packages/core/src/interpreter.ts

```diff
@@ -99,6 +99,7 @@ export class Interpreter<
   private delayedEventsMap: Record<string, unknown> = {};
 
   private observers: Set<Observer<SnapshotFrom<TBehavior>>> = new Set();
+  private stopListeners: Set<() => void> = new Set();
```

Comment:
was adding this intentional? currently, there is no way to add anything to this set

Reply:
It was not IIRC